11_FFTBloom/app_resources/common.hlsl

```diff
@@ -0,0 +1,76 @@
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+#define CHANNELS 3
```

Comment:
look at `ConstevalParameters` in `bindless_blit` branch https://github.com/Devsh-Graphics-Programming/Nabla/blob/2bca4ab30403748e9899790779b519dfe8b7c7cd/src/nbl/video/utilities/CComputeBlit.cpp#L54

I really dislike preprocessor stuff for anything but codegenning template code / specializations